graphql/types.lua

```diff
@@ -201,7 +203,10 @@ end
 
 types.int = types.scalar({
   name = 'Int',
-  description = "The `Int` scalar type represents non-fractional signed whole numeric values. Int can represent values between -(2^31) and 2^31 - 1. ", 
+  description = [[
+    The `Int` scalar type represents non-fractional signed whole numeric values.
+    Int can represent values between -(2^31) and 2^31 - 1.
+  ]],
```

Comment:
This adds two newlines into the string. Is it by intention?

Reply:
Thanks, made some "strips" and "gsubs" in the last commit